src/lib.rs

```diff
@@ -311,6 +311,7 @@ impl<T, F, P> AtomicPtr<T, F, P> {
 ///
 /// This type has the same in-memory representation as a [`std::ptr::NonNull`](core::ptr::NonNull).
 #[repr(transparent)]
+#[derive(Debug)]
```

Comment:
Needed for an unwrap in the new test

Reply:
We should manually implement `Debug` here then, since `derive(Debug)` will add an implicit `T: Debug` bound that's not necessary.